cos_registration_server/devices/models.py

```diff
@@ -15,6 +15,11 @@ def default_dashboards_json_field():
     return []
 
 
+def default_layouts_json_field():
```

Comment:
Since `Foxglove layouts should be passed with a name`, what's this default layout name? Can it be overwritten?

Reply:
The layout name is only for the COS registration server. The name is never passed to foxglove. It's only for the user to figure out what layout to use.
Foxglove json layouts don't have names, this is why the data structure for grafana_dashboards and foxglove_layout is different in the cos-registration-server